frontend/src/main/components/UCSBSubjects/UCSBSubjectsTable.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import OurTable, { ButtonColumn } from "main/components/OurTable";
+// import { toast } from "react-toastify";
+import { useBackendMutation } from "main/utils/useBackend";
+import { cellToAxiosParamsDelete, onDeleteSuccess } from "main/utils/UCSBSubjectUtils"
+import { useNavigate } from "react-router-dom";
+import { hasRole } from "main/utils/currentUser";
+
+export default function UCSBSubjectsTable({ subjects, currentUser }) {
+
+    const navigate = useNavigate();
+
+    const editCallback = (cell) => {
+        navigate(`/UCSBsubjects/edit/${cell.row.values.id}`)
```

Comment:
Capitalize the s in `UCSBsubjects` to match the controller. Should be `UCSBSubjects`
![image](https://user-images.githubusercontent.com/49248229/157384695-0c69720e-bef0-49fa-9b17-ae4e7193b19c.png)